src/components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import Logo from "../icons/logos/Logo"
+import NavbarLogin from "./NavbarLogin"
+import NavbarLogout from "./NavbarLogout"
+
+function Navbar(props){
+  return (
+    <header className="navbar--header">
+      <span className="navbar--logo">
+        <Link to="/">
+          <Logo />
+        </Link>
+      </span>
+      {console.log(props.setLoggedin)}
```

Comment:
Good practise is to not push console.log, or any useless code at all. Be sure to remove them when you finish working with them.